ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -107,6 +107,30 @@ test-suite test
                      , ouroboros-consensus-shelley-test
                      , ouroboros-consensus-cardano
 
+executable db-analyser
+  hs-source-dirs:      tools/db-analyser
+  main-is:             Main.hs
+  build-depends:       aeson
+                     , base
+                     , bytestring
+                     , cardano-binary
+                     , cardano-crypto-wrapper
+                     , cardano-ledger
+                     , cardano-slotting
+                     , containers
+                     , mtl
+                     , optparse-applicative
+                     , shelley-spec-ledger
+                     , split
+                     , time
+
+                     , ouroboros-consensus
+                     , ouroboros-consensus-byron
+                     , ouroboros-consensus-cardano
+                     , ouroboros-consensus-shelley
+                     , ouroboros-network
+  other-modules:
+                       Analysis
   default-language:    Haskell2010
   ghc-options:         -Wall
                        -Wcompat
```

Comment:
Those options belonged to the testsuite, now they are part of `executable db-analyser`. Copy them.